apps/api/src/domain/models/audio-visual/shared/commands/transcripts/translate-line-item/line-item-translated.event-handler.spec.ts

```diff
@@ -0,0 +1,137 @@
+        inPointMilliseconds: inPoint,
+        outPointMilliseconds: outPoint,
+    },
+});
+
+const existingAudioItem = buildTestInstance(EventSourcedAudioItemViewModel, {
+    id: audioItemId,
+    transcript: buildTestInstance(Transcript, {
+        participants: [participant],
+        items: [
+            buildTestInstance(TranscriptItem, {
+                inPointMilliseconds: inPoint,
+                outPointMilliseconds: outPoint,
+                text: buildMultilingualTextWithSingleItem(
+                    'existing transcript item text',
+                    originalLanguageCode
+                ),
+            }),
+        ],
+    }),
+});
+
+describe(`LineItemTranslatedEventHandler`, () => {
+    let databaseProvider: ArangoDatabaseProvider;
+
+    let app: INestApplication;
+
+    let testRepositoryProvider: IQueryRepositoryProvider;
+
+    let lineItemTranslatedEventHandler: LineItemTranslatedEventHandler;
+
+    beforeAll(async () => {
+        const moduleRef = await Test.createTestingModule({
+            imports: [PersistenceModule.forRootAsync()],
+        })
+            .overrideProvider(ConfigService)
+            .useValue(
+                buildMockConfigService(
+                    {
+                        ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+                    },
+                    buildConfigFilePath(Environment.test)
+                )
+            )
+            .compile();
+
+        await moduleRef.init();
+
+        app = moduleRef.createNestApplication();
+
+        const connectionProvider = app.get(ArangoConnectionProvider);
+
+        databaseProvider = new ArangoDatabaseProvider(connectionProvider);
+
+        testRepositoryProvider = app.get(QUERY_REPOSITORY_PROVIDER_TOKEN);
+
+        lineItemTranslatedEventHandler = new LineItemTranslatedEventHandler(
+            // @ts-expect-error TODO find a better way to sidestep this type issue
+            testRepositoryProvider
+        );
+    });
+
+    afterAll(async () => {
+        databaseProvider.close();
+    });
+
+    beforeEach(async () => {
+        await databaseProvider.clearViews();
+    });
+
+    describe(`when handling a LINE_ITEM_TRANSLATED`, () => {
+        beforeEach(async () => {
+            await testRepositoryProvider
+                .forResource(ResourceType.audioItem)
+                .create(existingAudioItem);
+        });
+
+        it(`should translate the line item`, async () => {
+            await lineItemTranslatedEventHandler.handle(lineItemTranslated);
+
+            const updatedView = (await testRepositoryProvider
+                .forResource(ResourceType.audioItem)
+                .fetchById(audioItemId)) as EventSourcedAudioItemViewModel;
+
+            const { text: foundMultilingualText } = updatedView.transcript.getLineItem(
+                inPoint,
+                outPoint
+            ) as TranscriptItem;
+
+            expect(foundMultilingualText.hasTranslation()).toBe(true);
```

Comment:
Add a note here that we do sanity checks for the handlers because the query repository test is comprehensive.

Reply:
done